exchanges/gateio/gateio_websocket_request_types.go

```diff
@@ -160,8 +160,8 @@ type WebsocketAmendOrder struct {
 // WebsocketFuturesAmendOrder defines a websocket amend order
 type WebsocketFuturesAmendOrder struct {
 	OrderID   string        `json:"order_id"`
-	Contract  currency.Pair `json:"-"` // This is not required in the payload, it is used to determine routing.
-	Asset     asset.Item    `json:"-"` // This is not required in the payload, it is used to determine routing.
+	Contract  currency.Pair `json:"-"` // Excluded from payload; used internally to determine routing
+	Asset     asset.Item    `json:"-"` // Excluded from payload; used internally to determine routing
```

Comment:
🐒 Pedantic monkey says that the first part of this sentence is self-evident
Commenting to say *why* is great though.
```suggestion
	Contract  currency.Pair `json:"-"` // Only used internally for routing
	Asset     asset.Item    `json:"-"` // Only used internally for routing
```